File: Moohvie/Models/moodpreset.swift
import Foundation

struct MoodPreset: Identifiable {
    let id: String
    let emoji: String
    let label: String
    let genreIDs: [Int]

    static let all: [MoodPreset] = [
        MoodPreset(id: "comedy",    emoji: "😂", label: "Comédie",    genreIDs: [35]),
        MoodPreset(id: "thrills",   emoji: "😱", label: "Frissons",   genreIDs: [27, 53]),
        MoodPreset(id: "emotion",   emoji: "🥹", label: "Émotion",    genreIDs: [18]),
        MoodPreset(id: "action",    emoji: "🚀", label: "Action",     genreIDs: [28, 12]),
        MoodPreset(id: "romance",   emoji: "💕", label: "Romance",    genreIDs: [10749]),
        MoodPreset(id: "scifi",     emoji: "🌌", label: "Imaginaire", genreIDs: [878, 14]),
    ]
}

File: android/app/src/main/java/noah/moohvie/models/MoodPreset.kt
package noah.moohvie.models

data class MoodPreset(
    val id: String,
    val emoji: String,
    val label: String,
    val genreIDs: List<Int>,
) {
    companion object {
        val all: List<MoodPreset> = listOf(
            MoodPreset("comedy",  "😂", "Comédie",    listOf(35)),
            MoodPreset("thrills", "😱", "Frissons",   listOf(27, 53)),
            MoodPreset("emotion", "🥹", "Émotion",    listOf(18)),
            MoodPreset("action",  "🚀", "Action",     listOf(28, 12)),
            MoodPreset("romance", "💕", "Romance",    listOf(10749)),
            MoodPreset("scifi",   "🌌", "Imaginaire", listOf(878, 14)),
        )

        fun find(id: String): MoodPreset? = all.firstOrNull { it.id == id }
    }
}
